data/remnant/remnant 2 side missions.txt

```diff
@@ -144,3 +144,224 @@ mission "Remnant: Visit to Paradise 1"
+	cargo "equipment" 18
+	passengers 11
+	landing
+	source "Caelian"
+	destination "Viminal"
+	to offer
+		has "Remnant: Shattered Light 5: done"
+	on offer
+		conversation
+			`On <destination> the spaceport is awash in conversation, and you spot snippets here and there of people talking about the recovery of the Shattered Light. Many are speculating as to what happened to it in the first place, while others are discussing the technical merits of what the research team did to recover it. Details travel quickly, it seems.`
+			`	The team you are sent to pick up are waiting next to the information desk. They have a collection of pallets nearby, and a small crowd of people gathered around in animated conversation. They spot you and people start waving their goodbyes as the group pick up their bags and head towards you.`
+			`	"Ah, <first>! Thank you for transport us. We are excited to see the Shattered Light and start studying this mystery you have found." They briskly head to your ship.`
+	on complete
+		event "remnant: shattered light"
+		event "remnant: shattered light research" 60
+		conversation
+			`The Remnant spend the trip busily discussing what is known so far and exchanging theories, eagerly asking you questions what you saw and reviewing your sensor records. By the time you reach Viminal, they have a whole collection of theories and have a preliminary plan of what studies they hope to do. When pressed, however, they all acknowledge that their top priority will be documenting the ship. Every scratch, every oddly charge component, every minor modification. "After all," they say, "knowing how it is now is the first step to determining how it arrived here."`
+			`	The entire complement of Remnant crowd the windows when you finally settled into final approach for the covered valley that houses the Shattered Light. You can see that the Remnant have been busy setting up large equipment towers around it, including several small encampments of tents. As you settle in for a landing, you see a camel team maneuver a sled with a large anti-aircraft battery into position, where it is promptly shrouded in a tent that blends into the rest of the camp.`
+			`	Traffic control directs you to a pad near a large door of the Shattered Light. Ruach and a small group of Remnant are gathered around a hologram of the ship, and seem to be co-ordinating teams exploring it. Large mounds of equipment appear to have expanded from the sides of the ship, some of which appears to be functional. As you go through your landing checks, the team thank you for the transportation and begins offloading their equipment.`
+			`	"Welcome back, <first>!" signs Ruach from across the pad. "We have made considerable progress already." She appears set to explain what they have discovered.`
+			choice
+				`	"Great, but please save your explanations for those who want them."`
```

Comment:
```suggestion
				`	"Great, but please spare me the details."`
```
Maybe a bit too rough? Could be something conveying the feeling "I don't follow science things" too